WEB_SERVICE/WEB/_HIGH/4XX_BYPASS/description.md

```diff
@@ -0,0 +1,19 @@
+
+# 4XX-Bypass
+
+The 4xx-bypass exploits refer to HTTP request fuzzing techniques that allow attackers to bypass server-side restrictions, accessing unauthorized resources or performing malicious actions on a web application. This can lead to unauthorized data access, privilege escalation, and other security risks.
+
+=== "Python example"
+  ```python
+    import requests
+
+    response = requests.get("http://www.some-url.com/unauthorized_path")
+
+    '''if we have some unauthorized path that gets us a 403 code,
+    we can try something like adding a query parameter to see if we can
+    trick the server by exploiting some mistake.'''
```

Comment:
would be nice to give an example on how the backend would handle this to show the vuln.

Reply:
@deadly-panda can you please check the example I provided